pkg/auth/routes.go

```diff
@@ -45,12 +45,18 @@ func NewRouter(p *deps.RootProvider, configSource *configsource.ConfigSource) *h
 		PathPattern: "/healthz",
 	}, p.RootHandler(newHealthzHandler))
 
+	router.Add(httproute.Route{
+		Methods:     []string{"GET"},
+		PathPattern: "/robots.txt",
+	}, p.RootHandler(newRobotsTXTHandler))
+
```

Comment:
Please also add this handler to
- `pkg/admin/routes.go`
- `pkg/resolver/routes.go`
- `pkg/images/routes.go`
- `pkg/portal/routes.go`

Reply:
Updated. However, only the main server will serve at the root, so search engines always find robots.txt at the root so they should always find main server's robots.txt.

Reply:
Removed according to discussion in slack: We need search engines to crawl them at least once so that they are removed from the index.